src/uct/ib/configure.m4

```diff
@@ -139,14 +149,27 @@ AS_IF([test "x$with_ib" = "xyes"],
        AC_CHECK_DECLS([IBV_CREATE_CQ_ATTR_IGNORE_OVERRUN],
                       [have_cq_io=yes], [], [[#include <infiniband/verbs.h>]])
 
+       AS_IF([test "x$with_mlx5" != xno], [
```

Comment:
Isn't with_mlx5 and with_mlx5_dv actually the same? What it the difference?

Reply:
fixed, removed `--with-mlx5`, replaced by `--with-mlx5-dv`. retested yes/guess/no with or without libmlx5.so found. retested rpm generation and dependencies.